draft-ietf-acme-acme.md

```diff
@@ -818,6 +814,30 @@ url (required, string):
 To fulfill this requirement, the ACME client should direct the user to the
 indicated web page.
 
+#### Applications List
```

Comment:
I agree that this is better as its own subsection, but it would go better under the "Registration Objects" section, since ultimately it's about how registrations refer to applications.